webpack/scenes/ContentViews/Details/ContentViewDetailActions.js

```diff
@@ -72,21 +71,21 @@ export const getContentViewRepositories = (cvId, params, status) => {
 export const getRepositoryTypes = () => get({
   type: API_OPERATIONS.GET,
   key: REPOSITORY_TYPES,
-  errorToast: error => __(`Something went wrong while retrieving the repository types! ${error}`),
+  errorToast: error => __(`Something went wrong while retrieving the repository types! Error: ${getResponseErrorMsgs(error.response)}`),
```

Comment:
I think explicitly saying `Error: ` isn't needed-- just in case you ever get an `error` with no `response` key, the text would make more sense without it.